From ad6f633ac408c4a7560dfde941644d9d7e5ab4e0 Mon Sep 17 00:00:00 2001
From: Michael Benedict <michaelbt@live.com>
Date: Mon, 3 Jun 2019 21:28:29 +1000
Subject: [PATCH 49/60] drivers: thermal: silence enum-conversion warnings

Signed-off-by: Michael Benedict <michaelbt@live.com>
---
 drivers/thermal/isp_cooling.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/thermal/isp_cooling.c b/drivers/thermal/isp_cooling.c
index e8b2227082a6..ec3e4a7be3e9 100644
--- a/drivers/thermal/isp_cooling.c
+++ b/drivers/thermal/isp_cooling.c
@@ -327,12 +327,12 @@ static int isp_set_cur_state(struct thermal_cooling_device *cdev,
 	return isp_apply_cooling(isp_device, state);
 }
 
-static enum tmu_noti_state_t isp_tstate = ISP_COLD;
+static enum isp_noti_state_t isp_tstate = ISP_COLD;
 
 static int isp_set_cur_temp(struct thermal_cooling_device *cdev,
 				bool suspended, int temp)
 {
-	enum tmu_noti_state_t tstate;
+	enum isp_noti_state_t tstate;
 
 	if (suspended || temp < EXYNOS_COLD_TEMP)
 		tstate = ISP_COLD;
-- 
2.17.1

